content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -153,6 +153,32 @@ section refers to several key workload enablers and how they map to Windows.
   * `emptyDir` volumes
   * Named pipe host mounts
   * Resource limits
+  * OS field: This `Name` field within OS field should be set to `windows` to indicate that the current pod is a Windows pod. 
+              Please note that there is no effect of having value for this field since it is behind a featuegate `IdentifyPodOS` and it is currently alpha.
+              If the `Name` field in `OS` field is set to `windows`, following fields must be unset on the Windows pods when the `IdentifyPodOS` featuregate is enabled:
```

Comment:
What's the name of the field that should be set to `windows`? (It's unlikely to be `OS.Name` in the API; the API usually uses camelCase aka lowerCamelCase).

Reply:
As you noted [here](https://github.com/kubernetes/website/pull/30436#discussion_r755351724), it is spec.os.name